src/networks.ts

```diff
@@ -418,4 +418,26 @@ export const networkInfo = new Map<number, NetworkInfo>([
       stagingBaseAPI: 'https://transaction-testnet.staging.safe.kava.io',
     },
   ],
+  [
+    660279,
+    {
+      chainID: 660279,
+      name: 'Xai',
+      shortName: 'xai',
+      currencySymbol: 'XAI',
+      baseAPI: 'https://transaction.safe.xai.games',
+      stagingBaseAPI: 'https://transaction-testnet.safe.xai.games',
+    },
+  ],
+  [
+    37714555429,
+    {
+      chainID: 37714555429,
+      name: 'Xai Testnet Sepolia',
+      shortName: 'Xai-testnet-sepolia',
```

Comment:
```suggestion
      shortName: 'xaitestnet',
```
The proper short name is `xaitestnet`
https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-37714555429.json#L4